recipes/ldraw/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set parts_version = "20190911" %}
```

Comment:
upstream and debian uses versions like `1902`

Reply:
Yeah, I saw an archlinux PKGBUILD that had the format above.

Just updated to use the debian versioning.